introduction.md

```diff
@@ -1,2 +1,8 @@
 Yo, what's up. My name is Gutz Von-berlichingen, and old mercenary who tries to learn cool stuffs.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?